frontend/main/components/MarketplacePage/Products.js

```diff
@@ -2,281 +2,312 @@
-      name: "20-Gallon Freshwater Tank Kit",
-      category: Category.Tanks,
-      description: "Complete tank kit for freshwater setups, including LED lighting and filtration system.",
-      average_rating: 4.7,
-      numreviews: 430,
-      price: 119.99
-    },
-    {
-      prodid: 12362,
-      sellername: "AquaticSolutions",
-      sellerid: 7907,
-      imgurl: "https://example.com/images/product12362.jpg",
-      name: "Marine Salt Mix (5kg)",
-      category: Category.Misc,
-      description: "Premium marine salt mix for creating the perfect saltwater environment.",
-      average_rating: 4.4,
-      numreviews: 220,
-      price: 24.99
-    },
-    {
-      prodid: 12363,
-      sellername: "AquariumSupplies",
-      sellerid: 7908,
-      imgurl: "https://example.com/images/product12363.jpg",
-      name: "Aquarium Filter Media (3-pack)",
-      category: Category.Misc,
-      description: "A set of 3 high-quality filter media replacements for optimal water filtration.",
-      average_rating: 4.6,
-      numreviews: 310,
-      price: 14.99
-    },
-    {
-      prodid: 12364,
-      sellername: "FishyWorld",
-      sellerid: 7909,
-      imgurl: "https://example.com/images/product12364.jpg",
-      name: "Swordtail Fish (Pack of 2)",
-      category: Category.Fish,
-      description: "Bright and active Swordtail fish, perfect for community aquariums.",
-      average_rating: 4.5,
-      numreviews: 250,
-      price: 11.99
-    },
-    {
-      prodid: 12365,
-      sellername: "AquaticHaven",
-      sellerid: 7910,
-      imgurl: "https://example.com/images/product12365.jpg",
-      name: "Aquarium Driftwood",
-      category: Category.Rocks,
-      description: "Natural driftwood for aquariums, perfect for providing hiding places and aesthetic appeal.",
-      average_rating: 4.8,
-      numreviews: 520,
-      price: 34.99
-    },
-    {
-      prodid: 12366,
-      sellername: "RockyReef",
-      sellerid: 7911,
-      imgurl: "https://example.com/images/product12366.jpg",
-      name: "Pumice Stone for Aquariums",
-      category: Category.Rocks,
-      description: "Natural pumice stones for aquariums, excellent for biological filtration and decoration.",
-      average_rating: 4.7,
-      numreviews: 300,
-      price: 18.99
-    },
-    {
-      prodid: 12367,
-      sellername: "PlantLovers",
-      sellerid: 7912,
-      imgurl: "https://example.com/images/product12367.jpg",
-      name: "Cryptocoryne Wendtii Plant",
-      category: Category.Plants,
-      description: "Easy-to-grow aquatic plant that thrives in low light and adds lush greenery to any tank.",
-      average_rating: 4.7,
-      numreviews: 470,
-      price: 11.99
-    }
+  {
+   // Clownfish
+    prodid: "1a2b3c4d5e",
+    sellername: "Ocean Wonders",
+    sellerid: "1234-5678",
+    imgurl: "https://miro.medium.com/v2/resize:fit:940/1*tqkPCH9gc8nv000FOpRoOA.png",
+    name: "Clownfish",
+    category: Category.SaltwaterFish,
```

Comment:
To make sorting by categories easier, there are only 5 categories: Fish, Rocks, Plants, Tanks, and Misc, so we will need to update our test data to use those categories.